packages/app/src/server/service/slack-command-handler/create-page-service.js

```diff
@@ -20,7 +20,7 @@ class CreatePageService {
     const normalizedPath = pathUtils.normalizePath(sanitizedPath);
 
     // generate a dummy id because Operation to create a page needs ObjectId
-    const dummyObjectIdOfUser = new mongoose.Types.ObjectId();
+    const dummyObjectIdOfUser = userChannelId || new mongoose.Types.ObjectId();
```

Comment:
Why is this change necessary?

Reply:
This is so that the page creator can also be converted based on slackID, if there is no slackID then the page creator will be 'Unknown'

Reply:
OK. But the type of `dummyObjectIdOfUser` must be type of User Document which will be like `{ _id: ObjectId }` at least.

Reply:
OK, so I think the variable name should be `userId`. Also, I want you to always use null check to compare if the object is truthy or not. So, the code should be like `userId != null ? userId : new mongoose.Types.ObjectId();`